core/src/main/java/org/elasticsearch/ingest/core/CompoundProcessor.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+//TODO(simonw): can all these classes go into org.elasticsearch.ingest?
```

Comment:
agreed, I guess we are not ready for that yet but we will do it at some point? If that is the plan we should still keep separated the classes that will eventually go out? This was the only purpose of the separate package.